notebooks/ch18/hyp_regression.md

```diff
@@ -0,0 +1,545 @@
+
+
+<div>
+<style scoped>
+    .dataframe tbody tr th:only-of-type {
+        vertical-align: middle;
+    }
+
+    .dataframe tbody tr th {
+        vertical-align: top;
+    }
+
+    .dataframe thead th {
+        text-align: right;
+    }
+</style>
+<table border="1" class="dataframe">
+  <thead>
+    <tr style="text-align: right;">
+      <th></th>
+      <th>occupation</th>
+      <th>type</th>
+      <th>income</th>
+      <th>education</th>
+      <th>prestige</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <th>0</th>
+      <td>accountant</td>
+      <td>prof</td>
+      <td>62</td>
+      <td>86</td>
+      <td>82</td>
+    </tr>
+    <tr>
+      <th>1</th>
+      <td>pilot</td>
+      <td>prof</td>
+      <td>72</td>
+      <td>76</td>
+      <td>83</td>
+    </tr>
+    <tr>
+      <th>2</th>
+      <td>architect</td>
+      <td>prof</td>
+      <td>75</td>
+      <td>92</td>
+      <td>90</td>
+    </tr>
+    <tr>
+      <th>...</th>
+      <td>...</td>
+      <td>...</td>
+      <td>...</td>
+      <td>...</td>
+      <td>...</td>
+    </tr>
+    <tr>
+      <th>42</th>
+      <td>janitor</td>
+      <td>bc</td>
+      <td>7</td>
+      <td>20</td>
+      <td>8</td>
+    </tr>
+    <tr>
+      <th>43</th>
+      <td>policeman</td>
+      <td>bc</td>
+      <td>34</td>
+      <td>47</td>
+      <td>41</td>
+    </tr>
+    <tr>
+      <th>44</th>
+      <td>waiter</td>
+      <td>bc</td>
+      <td>8</td>
+      <td>32</td>
+      <td>10</td>
+    </tr>
+  </tbody>
+</table>
+<p>45 rows × 5 columns</p>
+</div>
+
+
+
+It is usually a good idea to explore the data through visualization in order to gain an understanding of the relationships between your variables. Below, we will visualize the correlations between income, educationa and prestige.
+
+
+```python
+plt.scatter(x=duncan["education"], y=duncan["prestige"])
+```
+
+
+
+
+    <matplotlib.collections.PathCollection at 0x1a1cf2cd30>
+
+
+
+
+![png](hyp_regression_files/hyp_regression_5_1.png)
+
+
+
+```python
+plt.scatter(x=duncan["income"], y=duncan["prestige"])
+```
+
+
+
+
+    <matplotlib.collections.PathCollection at 0x1a1d0224e0>
+
+
+
+
+![png](hyp_regression_files/hyp_regression_6_1.png)
+
+
+
+```python
+plt.scatter(x=duncan["income"], y=duncan["education"])
+```
+
+
+
+
+    <matplotlib.collections.PathCollection at 0x1a1d0de5f8>
+
+
+
+
+![png](hyp_regression_files/hyp_regression_7_1.png)
+
+
+From the plots above, we see that both education and income are positively correlated with prestige; hence, both of these variables might be useful in helping explain prestige. Let's fit a linear model using these explanatory variables to explain prestige.
+
+### Fitting the model 
+
+We will create the following model, that explains the prestige of an occupation as a linear function of income and education:
+
+$$
+\texttt{prestige}_i
+= \theta_0^* 
++ \theta_\texttt{income}^*\cdot\texttt{income}_i
++ \theta_\texttt{education}^*\cdot\texttt{education}_i
++ \varepsilon_i
+$$
```

Comment:
Can you try wrapping the math in a `\begin{aligned}` environment? The math isn't showing up for me in nbviewer.

```latex
$$
\begin{aligned}
\texttt{prestige}_i
= \theta_0^* 
+ \theta_\texttt{income}^*\cdot\texttt{income}_i
+ \theta_\texttt{education}^*\cdot\texttt{education}_i
+ \varepsilon_i
\end{aligned}
$$
```

Also, is there a reason you use \theta^* here instead of \hat{\theta}?